cni/network/network.go

```diff
@@ -252,10 +313,6 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 		epInfo.Routes = append(epInfo.Routes, network.RouteInfo{Dst: route.Dst, Gw: route.GW})
 	}
 
-	// Populate DNS info.
-	epInfo.DNS.Suffix = result.DNS.Domain
```

Comment:
check network config and if dns is specified, override epinfo dns otherwise use whatever ipam returns

Reply:
done.